tests/unit/nmk_unit_test_case.py

```diff
@@ -1,12 +1,14 @@
 # -*- coding: utf-8 -*-
 
-from django.test import TestCase, Client
+from django.db import connection, connections, DEFAULT_DB_ALIAS
+from django.test import TransactionTestCase, Client
+from django.test.utils import CaptureQueriesContext
 from django.urls import reverse
 
 from nmkapp import logic, views, cache, models
 
 
-class NmkUnitTestCase(TestCase):
+class NmkUnitTestCase(TransactionTestCase):
```

Comment:
TestCase nasledjuje TransactionTestCase, tako da moze da ostane TestCase, jel to ok?